_posts/en/newsletters/2021-05-05-newsletter.md

```diff
@@ -0,0 +1,176 @@
+    against the idea, [saying][russell backups] "I see encrypted backups
+    as a more-likely-to-be-implemented solution though.  Because they're
+    useful to send to places other than peers, and they can also contain
+    [HTLC][topic htlc] information if you want."  Being able to contain
+    [HTLC][topic htlc] information would allow settling payments that
+    were pending at that time, which is a capability no recovery
+    mechanism based solely on a BIP32 seed could provide.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.21.1][Bitcoin Core 0.21.1] is a new version of Bitcoin
+  Core that contains activation logic for the proposed [taproot][topic
+  taproot] soft fork.  Taproot uses [schnorr signatures][topic schnorr
+  signatures] and allows the use of [tapscript][topic tapscript].  These
+  are, respectively, specified by BIPs [341][BIP341], [340][BIP340], and
+  [342][BIP342].  Also included is the ability to pay [bech32m][topic
+  bech32] addresses specified by [BIP350][], although bitcoins spent to
+  such addresses on mainnet will not be secure until activation of a
+  soft fork using such addresses, such as taproot.  The release
+  additionally includes bug fixes and minor improvements.
+
+    Note: due to a [problem][wincodesign] with the certificate authority
+    that provides the code signing certificates for the Windows versions
+    of Bitcoin Core, users on Windows will need to click through an
+    extra prompt to install.  It is expected that there will be a
+    0.21.1.1 release with an updated certificate when the problem is
+    fixed.  If you are planning to upgrade anyway, there's no reason to
+    delay using 0.21.1 because of this problem.
+
+- [BTCPay 1.1.0][] is the latest major release for this self-hosted payment
+  processing software. It includes [Lightning Loop][news53 lightning loop]
+  support, adds [WebAuthN/FIDO2][fido2 website] as a two-factor authentication
+  option, makes various UI improvements, and marks a switch to using [semantic
+  versioning][semantic versioning website] for version numbers moving forward.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #19160][] is the next PR in the [multiprocess project][bitcoin
+  core multiprocess] and adds the ability for the `bitoin-node` process to spawn
+  other processes and communicate with them using [Capn' Proto][capn proto].
```

Comment:
```suggestion
  other processes and communicate with them using [Cap’n Proto][capn proto].
```
(noticed while opening the file; haven't read the rest yet and will try to do a full read-through a bit later)